lib/manageiq/environment.rb

```diff
@@ -0,0 +1,108 @@
+require 'fileutils'
+require 'pathname'
+
+module ManageIQ
+  module Environment
+    APP_ROOT = Pathname.new(File.expand_path('../../..', __FILE__))
+
+    def self.ensure_config_files
+      config_files = {
+        "certs/v2_key.dev"        => "certs/v2_key",
+        "config/cable.yml.sample" => "config/cable.yml",
```

Comment:
@Fryguy sure, should I create a PR or you'll do that? Don't forget to edit the `bin/setup` file too!